BeagleBone/AI/pru/blinkR30.c

```diff
@@ -0,0 +1,40 @@
+////////////////////////////////////////
+//	blinkR30.c
+//	Blinks LEDs wired to P9_25 (and others) by writing register R30 on the PRU
```

Comment:
We might want to throw a wiring diagram into the repo. I tried to do so with PocketBeagle, though it might be possible to do better: https://github.com/beagleboard/cloud9-examples/blob/master/PocketBeagle/PocketBeagle-examples1_bb.png